scripts/prepdocs.sh

```diff
@@ -58,6 +58,10 @@ if [ "$USE_VECTORS" = false ]; then
   disableVectorsArg="--novectors"
 fi
 
+if [ "$OPENAI_EMB_DIMENSIONS" = false ]; then
```

Comment:
= false or present?

Reply:
also, aren't you supposed to add dimensions as an actual argument here? Looks like you are passing it like a flag

Reply:
yes! I wonder if I changed this after testing. Will do another test run.

Reply:
I think I must have been accidentally using 1536 in my testing, not passing on the env vars, as I uncovered more errors when fixing this. 

That does bring up an issue that @tonybaloney mentioned in a meeting earlier- in theory someone could change all the variables, *not* re-index any data, and then discover absolute nonsense search results in the app, because the app query model doesn't match the indexed model. We could add a field to the index to track the model used, but we really don't want to add fields in this repo since that isn't backwards compatible. I am okay with that being a possible situation, given that people have to read the README anyway in order to set this up, and it says re-indexing is needed. But wanted to mention it.